Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/Http.rst

```diff
@@ -205,52 +205,25 @@ still handled even if the new component cancels the current chain.
-	unexpected results, like the trusted proxy headers ``X-Forwarded-*`` not being applied and the ``Request`` providing
-	wrong protocol, host or client IP address.
-	If you need access to the **current** HTTP ``Request`` or ``Response``, instead inject the ``Bootstrap`` and
-	get the ``HttpRequest`` and ``HttpResponse`` through the ``getActiveRequestHandler()``.
-  Note though that those two objects are immutable, so if you need to change them, you need to in turn tell the current
-  ``ComponentContext`` that they need to be replaced through the ``->replaceHttpRequest(..)`` and ``->replaceHttpResponse(..)``
-  methods respectively. You can get the current ``ComponentContext`` from the active ``RequestHandler``.
-
-create()
-~~~~~~~~
-
-The method ``create()`` accepts an URI, the request method, arguments and a few more parameters and returns a new
-``Request`` instance with sensible default properties set. This method is best used if you need to create a new
-``Request`` object from scratch without taking any real HTTP request into account.
-
-createFromEnvironment()
-~~~~~~~~~~~~~~~~~~~~~~~
-
-The second method, ``createFromEnvironment()``, take the environment provided by PHP's superglobals and specialized
-functions into account. It creates a ``Request`` instance which reflects the current HTTP request received from the web
-server. This method is best used if you need a ``Request`` object with all properties set according to the current
-server environment and incoming HTTP request.
-Note though, that you should not expect this ``Request`` to match the current ``Request``, since the latter will still
-have been affected by some ``HTTP Components``. If you need the **current** Request, get it from the ``RequestHandler`` instead.
+In the PSR-7 specification, a distinction is made between two different types of requests - incoming (``ServerRequest``)
+and outgoing (``Request``). Whenever you want to make an outgoing request, you can easily use for example Guzzle's
+``Request`` class constructor with the respective arguments for method, uri, etc. and then pass that to e.g. a PSR-18
+Http Client implementation.
+On the other side the incoming request is something you should never try to create an instance of yourself, as it is
+provided by the framework. In theory, you could also call inject the ``ServerRequestFactory::createServerRequest`` or
+Guzzle's ``ServerRequest::fromGlobals()`` convenience method, but this does not have any relation to the current request
+object handled by the framework. It will not have any of the processing from components applied and might therefore lead
+to unexpected results, like the trusted proxy headers ``X-Forwarded-*`` not being applied and the ``ServerRequest`` providing
+wrong protocol, host or client IP address.
+If you need access to the **current** HTTP ``Request``, either create a Http Component or only access it inside the
+controller through the ``ActionRequest`` for inspecting.
```

Comment:
A link to the HTTP Component section and a mini example for the ActionRequest case would be rad